client/my-sites/plan-features-2023-grid/style.scss

```diff
@@ -416,6 +416,45 @@ $mobile-card-max-width: 440px;
 		}
 	}
 
+	.plan-features-2023-grid__current-plan {
+		.plan-features-2023-grid__table.plan-features-2023-grid__current-plan-card {
+			max-width: 100%;
+			margin-bottom: 64px;
+			td.plan-features-2023-grid__table-item {
+				&.is-top-buttons {
+					position: absolute;
+					top: 46px;
+					right: 0;
+				}
+
+				&.plan-features-2023-grid__header-billing-info {
+					padding-bottom: 32px;
+				}
+
+				.plan-features-2023-grid__popular-badge {
+					padding-bottom: 23px;
+				}
+
+				.plan-features-2023-grid__header-title {
+					margin-top: 6px;
+				}
+
+				.plan-features-2023-grid__header-tagline {
+					@include plans-2023-break-medium {
+						min-height: 0;
+						padding-bottom: 16px;
+					}
+				}
+
+				.plan-features-2023-grid__header-logo {
+					@include plans-2023-break-small {
+						margin-top: 18px;
+					}
+				}
```

Comment:
Is it 100% needed to have these media queries here? I'd love to see us rely less and less on these, even if it means sacrificing a few pixels top/bottom. Most likely these media queries won't be used at all for anything under `plan-features-2023-grid`, probably not in their current form at least (which will require a hefty refactor).

Just asking - can you achieve this without?

Reply:
Good catch! Those weren't needed at all. I have removed them in https://github.com/Automattic/wp-calypso/pull/78845/commits/b4d2d720779250e28504fe442420cae3734e32c9.